lib/compat/wordpress-6.2/block-patterns.php

```diff
@@ -18,50 +18,57 @@ function gutenberg_register_core_block_patterns_and_categories() {
 	register_block_pattern_category(
 		'buttons',
 		array(
-			'label' => _x( 'Buttons', 'Block pattern category', 'gutenberg' ),
+			'label'       => _x( 'Buttons', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Patterns that contain buttons and call to actions.', 'gutenberg' ),
 		)
 	);
 	register_block_pattern_category(
 		'columns',
 		array(
-			'label' => _x( 'Columns', 'Block pattern category', 'gutenberg' ),
+			'label'       => _x( 'Columns', 'Block pattern category', 'gutenberg' ),
+			'description' => __( 'Multi column patterns with more complex layouts.', 'gutenberg' ),
```

Comment:
Should we use a hyphen for creating a compound word here - "Multi-column"?

Sorry, I can't remember the exact grammar rule.

Reply:
Sounds right to me with the hyphen.

Reply:
Thanks! I updated it.